src/solver/conv_ocl_dir2Dfwd_exhaustive_search.cpp

```diff
@@ -274,8 +276,8 @@ ConvOclDirectFwdLegacyExhaustiveSearch::SearchImpl(const ConvolutionContext& ctx
     std::vector<Tgpu> wei_sys_buf(weights_sz);
     for(size_t i = 0; i < weights_sz; i++)
     {
-        wei_sys_buf[i] = static_cast<Tgpu>((rand() * (1.0 / RAND_MAX) - 0.5) *
-                                           0.001); // NOLINT (concurrency-mt-unsafe)
+        wei_sys_buf[i] =
+            static_cast<Tgpu>((std::generate_canonical<double, 11>(prng) - 0.5) * 0.001);
```

Comment:
Why not use `miopen::prng` variant ?

Reply:
I'm not sure about including driver headers into solver. If it's ok, then `miopen::prng` is the choise.

Reply:
I see your point, perhaps move the header to MIOpen and then use it both in the solver and the driver